<% if @task.errors.any? %>
<div id="error_explanation">
  <h2><%= @task.errors.count %>件のエラーがあります。</h2>
  <ul>

  <% @task.errors.full_messages.each do |msg| %>
    <li><%= msg %></li>
  <% end %>
  </ul>
</div>
<% end %>
<%= form_with(model: @task, local: true) do |form| %>
  <div class="task_substance">
    <%= form.label :タスク名 %>
    <%= form.text_field :substance %>
  </div>
  <div class="task_content">
    <%= form.label :詳細 %>
    <%= form.text_field :content %>
  </div>
  <div class="task_period">
  <%= form.label :終了期限 %>
  <%= form.datetime_field :period %>
</div>
<div class="task_state_for_progress">
<%= form.label :ステータス %>
<%= form.select :state_for_progress, [["未着手", "未着手"], ["着手中", "着手中"], ["完了", "完了"]], include_blank: "選択して下さい" %>
</div>
  <%= form.submit %>
<% end %>

<%= link_to '一覧画面に戻る', tasks_path %>

